### ISSUE TITLE
OnButtonHeld event broken

### ISSUE THREAD

**Issue by Rex-22** [GameFoundry/bsf#309]
The OnButtonHeld event for virtual input is never fired. 

Here is a example from the docs, but it does not work as expected.

```c++
	SPtr<InputConfiguration> inputConfig = gVirtualInput().getConfiguration();
	VirtualButton forwardKey("Forward");

	// Register a virtual button named "Forward" that maps to W and Up arrow keys
	inputConfig->registerButton("Forward", BC_W);
	inputConfig->registerButton("Forward", BC_UP);

	// Callback method that triggers when any virtual button is being held down
	auto handleButtonHeld = [&](const VirtualButton& btn, UINT32 deviceIdx)
	{
		// If user holds down W or Up arrow, move forward
		if (btn == forwardKey){
			LOGWRN("Hello there")
		}
			
	};
	// Connect the callback to the event
	gVirtualInput().onButtonHeld.connect(handleButtonHeld);
``` 

The code should print out `Hello there` but there is no output.

I did some digging in the source and the onButtonHeld event is never called from what I can see.

https://github.com/GameFoundry/bsf/blob/f3a903b42770716be643b57ce6135d785ded59f9/Source/Foundation/bsfEngine/Input/BsVirtualInput.h#L107

**Comment by BearishSun** [comment]
Thanks for the report. I'll let you know when I take a closer look.

**Comment by BearishSun** [comment]
This should be resolved in 48e8189da99722db189c757064cc3d3c0acf1686.
